src/Common/src/Interop/OSX/Interop.EventStream.cs

```diff
@@ -49,7 +50,10 @@ internal enum FSEventStreamEventFlags : uint
             kFSEventStreamEventFlagItemXattrMod         = 0x00008000,
             kFSEventStreamEventFlagItemIsFile           = 0x00010000,
             kFSEventStreamEventFlagItemIsDir            = 0x00020000,
-            kFSEventStreamEventFlagItemIsSymlink        = 0x00040000
+            kFSEventStreamEventFlagItemIsSymlink        = 0x00040000,
+            kFSEventStreamEventFlagOwnEvent             = 0x00080000,
+            kFSEventStreamEventFlagItemIsHardlink       = 0x00100000,
+            kFSEventStreamEventFlagItemIsLastHardlink   = 0x00200000,
```

Comment:
No, but they're very useful when debugging the FSW since otherwise we get bit values and not the names of what happened